src/storybook-addons/storyWrapper.test.js

```diff
@@ -0,0 +1,61 @@
+/* global describe, expect, it, mount  */
+import React from 'react';
+import PropTypes from 'prop-types';
+import storyWrapper from './storyWrapper';
+
+const MockComponent = props => (<div>{props.name}</div>);
+MockComponent.propTypes = {
+    name: PropTypes.any, // eslint-disable-line react/forbid-prop-types
+};
+MockComponent.defaultProps = {
+    name: '',
+};
+
+function prepareWrapper() {
+    const hoc = storyWrapper(
+        'My summary',
+        <MockComponent name="example" />,
+        <div>
+            <MockComponent name="example" />
+        </div>,
+    );
+    const storyContextMock = { kind: 'Foo', story: 'Bar' };
+    return mount(hoc(storyContextMock));
+}
+
+describe('storyWrapper', () => {
+    it('renders our logo', () => {
+        const wrapper = prepareWrapper();
+        expect(wrapper.find('svg').length).to.equal(1);
```

Comment:
This doesn't really test if it renders *our* logo, just a logo. Perhaps that's ok, if so the test title could be changed to "renders a logo".
Or if you want to be more thorough, you could add a proper `<title>` to the svg (currently it has some dummy text) and test for that. And you can also check the `fill` attribute on the path to see that it uses the correct brand colour (#9a3cbb).
But I'm not sure if we need to be this thorough.

Reply:
Very good point, I've added some tests to be thorough :)